server/src/test/java/io/druid/server/coordination/ZkCoordinatorTest.java

```diff
@@ -283,4 +291,84 @@ private void checkCache(List<DataSegment> segments) throws IOException
       Assert.assertEquals(segments.get(i), segment);
     }
   }
+  
+  @Test
+  public void testInjector() throws Exception
+  {
+    Injector injector = Guice.createInjector(new Module() {
```

Comment:
Usually injectors are created in a method similar to https://github.com/druid-io/druid/blob/master/server/src/test/java/io/druid/client/cache/MemcachedCacheTest.java#L144  

where 
```binder.bindConstant().annotatedWith(Names.named("serviceName")).to("druid/test/memcached");
binder.bindConstant().annotatedWith(Names.named("servicePort")).to(0);```
is used inside of a call to `Initialization.makeInjectorWithModules( GuiceInjectors.makeStartupInjector(), ImmutableList.of(`

Is that method able to be used here?